indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/ParallelIndexSupervisorTask.java

```diff
@@ -1633,6 +1633,14 @@ private Pair<Map<String, Object>, Map<String, Object>> doGetRowStatsAndUnparseab
     );
     buildSegmentsRowStats.addRowIngestionMetersTotals(rowStatsForRunningTasks);
 
+    // Emit the processed bytes metric
+    try {
+      emitMetric(toolbox.getEmitter(), "ingest/processed/bytes", rowStatsForRunningTasks.getProcessedBytes());
+    }
+    catch (Exception e) {
```

Comment:
I don't think you need a try-catch here. `rowStatsForRunningTasks` should always be non-null afaict.